nncf/quantization/algorithms/fast_bias_correction/onnx_backend.py

```diff
@@ -96,12 +99,31 @@ def create_blob(shape: Tuple[int], data: List[float]) -> np.ndarray:
         return blob
 
     @staticmethod
-    def get_initializer_value(model: onnx.ModelProto, initializer_name: str) -> np.ndarray:
-        for initializer in model.graph.initializer:
-            if initializer.name == initializer_name:
-                return onnx.numpy_helper.to_array(initializer)
-        raise RuntimeError('There is no initializer with the name {}'.format(initializer_name))
+    def get_bias_value(model: onnx.ModelProto, node: NNCFNode) -> np.ndarray:
+        onnx_graph = ONNXGraph(model)
+        node = onnx_graph.get_node_by_name(node.node_name)
+        # We uses 2nd value from the tensor names
+        # because of the bias tensor placement on this position.
```

Comment:
Is the bias tensor really placed ONLY on this position? Is either for `Linear` and `MatMul` layers?

Reply:
For simplification we stay with that. When we will extend the model coverage, this should be also updated with the new logic, where bias value is not tied to the exact index.

Reply:
But is 2nd value really valid for the layers mentioned above? I mean, not only do `Convolutions` contains `Biases`, but the other layers can be found and checked in common models.

Reply:
Layers that contain biases are lay in the List LAYERS_WITH_BIAS_METATYPES. For now there are only Convolution and TransposeConvolution. So we searching for biases only for the,

Reply:
Okay, I see that these types are not important now.